indico/modules/designer/controllers.py

```diff
@@ -177,9 +177,10 @@ def _checkProtection(self):
             raise Forbidden
 
     def _process(self):
-        form = AddTemplateForm()
+        form = AddTemplateForm(target=self.target)
         if form.validate_on_submit():
-            new_template = DesignerTemplate(title=form.title.data, type=form.type.data, **self.target_dict)
+            new_template = DesignerTemplate(title=form.title.data, type=form.type.data,
+                                            is_clonable=form.is_clonable.data, **self.target_dict)
```

Comment:
accessing `form.is_clonable.data` is likely to fail when you deleted the field from the form.
I'd just do something like this after this statement:

    if 'is_clonable' in form and not form.is_clonable.data:
        new_template.is_clonable = False